pkgs/development/libraries/qt-6/qtModule.nix

```diff
@@ -40,7 +40,9 @@ stdenv.mkDerivation (
 
     cmakeFlags =
       args.cmakeFlags or [ ]
-      ++ lib.optional stdenv.hostPlatform.isDarwin "-DQT_NO_XCODE_MIN_VERSION_CHECK=ON";
+      ++ lib.optional stdenv.hostPlatform.isDarwin "-DQT_NO_XCODE_MIN_VERSION_CHECK=ON"
+      # https://bugreports.qt.io/browse/QTBUG-136060
+      ++ [ "-DCMAKE_SYSTEM_VERSION=" ];
```

Comment:
This is like, extremely nitpicky, but I'd maybe flip the order of these?

```
cmakeFlags = [ "-DCMAKE_SYSTEM_VERSION=" ]
  ++ lib.optional stdenv.hostPlatform.isDarwin "-DQT_NO_XCODE_MIN_VERSION_CHECK=ON"
  ++ args.cmakeFlags or [];
```

just scans better to me?